utils/weight-diff/src/main.rs

```diff
@@ -266,5 +340,102 @@ fn main() {
             println!("{table}");
             println!();
         }
+        Commands::Codegen { path, runtime } => {
+            let dump: DeserializableDump =
+                serde_json::from_str(&fs::read_to_string(path).unwrap()).unwrap();
+            let raw_schedule = match runtime {
+                Runtime::Vara => serde_json::to_value(dump.vara_schedule).unwrap(),
+            };
+
+            let file =
+                syn::parse_file(&fs::read_to_string("pallets/gear/src/schedule.rs").unwrap())
+                    .unwrap();
+
+            let mut visitor = StructuresVisitor::default();
```

Comment:
Could you please clarify the purpose of visitor if all structure names are `DeserializableSchedul`'s fields

Reply:
Because we would like to keep the original data types for structure fields.